src/style.css

```
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 2px 0 0 2px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin: 0 7px 4px 0;
```

```suggestion
  margin-right: 4px;
```
Тут також потрібно додати background-image, width, height, background-position, background-repeat;

---

ссилки чомусь не працюють 
npm run deploy зробила 
тести запускала і лінт також 

- [DEMO LINK
- https://vladyslava-buzova.github.io/layout_stars/
- [TEST REPORT LINK]
- https://vladyslava-buzova.github.io/layout_stars/report/html_report/